core/src/main/scripts/importer/validateData.py

```diff
@@ -643,6 +643,33 @@ def checkGeneIdentification(self, gene_symbol=None, entrez_id=None):
         # set to upper, as both maps contain symbols in upper
         if gene_symbol is not None:
             gene_symbol = gene_symbol.upper()
+            # Check in case gene symbol is not null if it starts with an integer
+            if gene_symbol is not '':
```

Comment:
We decided to keep it in, since validation for other file types do not set empty values to `none`, and it is out of scope for this PR to refactor that.